### PR TITLE
Eliane's Pizza Project - Week 1

### PR DESCRIPTION
**Pull request by El1an3**
## Netlify link
https://snazzy-dango-e4f6f2-pizza-project.netlify.app/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (AntonellaMorittu) [inline .gitignore:1]: .DS_Store is a file that stores custom attributes of its containing folder, such as folder view options, icon positions, and other visual information, and we don't need to share it in our project. Please add .DS_Store to your .gitignore file 👍

File: README.md
Comment (AntonellaMorittu) [inline README.md:1]: Nice to document your project and update your `README.md` 😎

File: code/index.html [hunk 2]
```diff
@@ -13,6 +13,8 @@
   <body>
     <h1>Javascript Pizzeria</h1>
     <p>Logic is executed automatically</p>
-    <script src="./script.js"></script>
+    <script src="./switch.js"></script>
   </body>
 </html>
+// just adding a few lines to see if I can push again and fix the netlify
```
Comment (AntonellaMorittu) [inline code/index.html:19]: make sure to remove this line, is showing in the UI 🧹

File: code/script.js [hunk 3]
```diff
@@ -2,18 +2,152 @@
...
+
+// Step 4
+// This part defines size of the meal based on the age of the person ordering
+let yourAge = prompt("Please enter your age to define the size of your meal:");
+
+let orderCost = 10;
+if (yourAge <= 12) {
+  yourAge = "child size";
+  alert(
+    "You ordered a " +
```
Comment (AntonellaMorittu) [inline code/script.js:114]: This works fine, but have you considered using template literals? They can help you save some characters and make the text look more readable, like so:
```
`You ordered a ${yourAge} ${foodChoice} for ${orderCost} EUR`
```

File: code/script.js [hunk 4]
```diff
@@ -2,18 +2,152 @@
...
+      " for " +
+      (orderCost + 5) +
+      " EUR! "
+  );
+} else {
+  alert("Invalid response. Please start over!");
+}
 
 // Step 5 - Order confirmation
-// Your code goes here
```
Comment (AntonellaMorittu) [inline code/script.js:19]: nice touch 😎

File: code/switch.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+    `Please enter your age to define the size of your meal.
+Children under the age of 13 get a child sized meal.`
+  )
+);
+
+let orderCost = 10;
+switch (true) {
+  case yourAge <= 12:
+    yourAge = "child size";
+    alert(`You ordered a ${yourAge} ${foodChoice} for ${orderCost} EUR!`);
```
Comment (AntonellaMorittu) [inline code/switch.js:142]: great you used it here! 🥂

**Review by AntonellaMorittu — APPROVED** [review]
Hi Eliane, you did great this week! 🍕  the code meets the requirements and stretch goals, that's brilliant! I left some comments specifically on some parts, please take a look at them :) Thank you ⭐

File: code/script.js (hunk 3 above)
Comment (El1an3) [reply code/script.js:114]: As I changed it all to literals in the switch.js, I'll leave it here like this. Just to have both options available.

**Pull request closed without merging**